.goreleaser.yml

```diff
@@ -41,11 +41,23 @@ builds:
       - darwin
     goarch:
       - amd64
+      - arm64
     hooks:
       post: |-
         sh -c '
-        zip ./dist/slack_cli_{{.Env.BUILD_VERSION}}_macOS_64-bit.zip ./dist/slack-macos_darwin_amd64_v1/bin/slack -j
+        zip ./dist/slack_cli_{{ .Env.BUILD_VERSION }}_macOS_{{ .Arch }}.zip ./dist/slack-macos_{{ .Target }}/bin/slack -j
         '
+
+universal_binaries:
+  - id: slack-macos
+    name_template: bin/slack
+    replace: false
+    hooks:
+      post: |-
+        sh -c '
+        zip ./dist/slack_cli_{{.Env.BUILD_VERSION}}_macOS_64-bit.zip ./dist/slack-macos_darwin_all/bin/slack -j
+        '
```

Comment:
📚 https://goreleaser.com/customization/universalbinaries/

The `name_template` is used to match packaging paths in tarballs later!